packages/api/README.md

```diff
@@ -1,5 +1,12 @@
 # API
 
+### Prerequisite files
+1. `.env`file. It contains two variables `HUB_MAIL_USERNAME` and `HUB_MAIL_PASSWORD` used for authentication when setting up Nodemailer in the `email.js` file.
```

Comment:
Swap the numbers for dots and display the example env file as follows:

```
ENV_VAR_1=<value>
ENV_VAR_2=<value>
```

Leave `<value>` as it is.